scripts/recipes/librispeech/run.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+set -euo pipefail
+
+source ./path.sh
+
+lm_url=www.openslr.org/resources/11
+work_dir=work_1a
+
+stage=3
+
+mkdir -p $work_dir
+
+
+# LD_PRELOAD=$(gcc -print-file-name=libasan.so) 
+# bazel build -c dbg --config=asan --cxxopt='-std=c++14'  --verbose_failures lingvo:trainer third_party/kaldi/... @openfst//... galvasr2/...
+# bazel build --cxxopt='-std=c++14' lingvo:trainer third_party/kaldi/... @openfst//... galvasr2/...
```

Comment:
@anjaligopi you would copy paste this line to build all of the prerequisites of the project now FYI.